advisor/index.go

```diff
@@ -578,6 +584,10 @@ func (idxAdv *IndexAdvisor) mergeIndexes(idxList []IndexInfo) []IndexInfo {
 	return rmSelfDupIndex(indexes)
 }
 
+func getIndexNameSuffix() string {
```

Comment:
Add comment, rename func name
// getRandomIndexSuffix format: _xxxx, length: 5
func getRandomIndexSuffix() string{

Reply:
Add test case for this function
TestGetRandomIndexSuffix
* length == 5
* HasPrefix '_'